### DISCUSSION TITLE
[node-schedule] new gracefulShutdown method and success event

### DISCUSSION THREAD

**Discussion by locomoco28** [DefinitelyTyped/DefinitelyTyped#59947] [Issues with a @types package]
`node-schedule` has gotten a new `gracefulShutdown` method with version `2.1.0`

See: [node-schedule/node-schedule#583](https://github.com/node-schedule/node-schedule/pull/583)

---

There is also a new `success` event emitted

See: [node-schedule/node-schedule#643](https://github.com/node-schedule/node-schedule/pull/643)

[a comment by an automated account is not shown]

**Comment by OmgImAlexis** [comment]
@locomoco28 What has this todo with node-fetch? I'm not seeing any references to it in that PR.

**Reply by locomoco28** [reply]
The types are missing the `gracefulShutdown` property, so I figured the latest `2.1.0` changes are not implemented yet @OmgImAlexis

**Reply by kyranet** [reply]
@locomoco28 your links are for `node-schedule`, but you mentioned `node-fetch`'s team.

**Reply by OmgImAlexis** [reply]
@locomoco28 wrong repo though?

**Reply by locomoco28** [reply]
Sorry yes, I meant node-schedule, not fetch 🤦‍♂️ 
Was updating my dependencies and mixed up the two
(Edited the issue)
